client/layout/style.scss

```diff
@@ -132,6 +132,11 @@
 
 .layout.focus-sidebar {
 	overflow: hidden;
+
+	.layout__primary {
+		opacity: 0.2;
+		pointer-events: none;
```

Comment:
We can't do display:none? Seems odd that we're still rendering this

Reply:
It does; I admit I'm going with the exact same handling that `.layout.focus-sites .layout__primary` has right above in the same file (and, counter-intuitively, failing any sort of DRY check in the process). That's why I'm suggesting this as just a quick fix consistent with current handling of the Sites view, and maybe not the best in the long run).

Reply:
@gwwar you're right, those styles are not at all intended for what I was using them for.

Reply:
Hmm I think this works because we've introduced another stacking context. Here it's the primary div (when sidebar is visible)
<img width="454" alt="screen shot 2018-02-08 at 4 10 05 pm" src="https://user-images.githubusercontent.com/1270189/36005087-c361e8fc-0cea-11e8-8540-268d11b7f045.png">

Versus root when we're in initial desktop view:
<img width="464" alt="screen shot 2018-02-08 at 4 12 41 pm" src="https://user-images.githubusercontent.com/1270189/36005131-fb368dc8-0cea-11e8-95ee-e9df4cfe2507.png">